### ISSUE TITLE
BI Template not aligned with April Project Blog referenced PBIT file

### ISSUE THREAD

**Issue by tullisbrian** [OfficeDev/Project-Accelerator#37]
Not sure if this is a feature or a bug. We are using Accelerator in production....and when we saw the April Project blog post about an updated PBIT file with direct links to tasks, we installed it. (ref: https://techcommunity.microsoft.com/t5/project-blog/april-project-update-blog/ba-p/2319011)

However, that PBIT file does not have any of the Accelerator entities such as Status, Risk, Issue, Program, etc.

It looks like we have to revert to the PBIT referenced in the Accelerator documentation. Will the Project team continue to maintain that PBIT file separate from this one? Can you merge your efforts? My preference would be for you to update the Accelerator PBIT with the direct-to-task link for Task Status as they did in the Project PBIT file....and of course keep all of the Accelerator entities in tact. Thanks.

**Comment by hmbouk** [comment]
I used the original PBIT to create my report that I treat as my master report for my projects, programs, etc.  As I see new PBIT files I will open them to see what new report pages might be available.  I will then copy any new pages into my file.  You might have to manually create the supporting tables first before copying the report page.  

For example I saw this blog post for a Resource Allocation report https://techcommunity.microsoft.com/t5/project-blog/monitor-resource-allocation-with-project-power-bi-template/ba-p/2492453 and followed these steps to add to my report.

It might be nice is the report efforts were coordinated but it isn't to hard to incorporate changes into your existing PBIX file.
